doc/qos/SONiC_ROCEv2_support.md

```diff
@@ -539,18 +1475,161 @@ sonic# show buffer interface Ethernet0 queue
 Interface       Queue          Profile
 Ethernet0        0             ingress_lossy_profile (edited)
 ```
-#### 3.6.2.3 Sample configurations
+
+
+#### 3.6.2.5 Debug Commands
+
+#### 3.6.2.6 IS-CLI Compliance
+
+### 3.6.3 REST API Support
+
+```
+GET - Get existing buffer configuration information from CONFIG DB.
+POST - Add buffer configuration into CONFIG DB.
+PATCH - Update existing buffer Configuration information in CONFIG DB.
+DELETE - Delete a existing buffer configuration from CONFIG DB.
+```
+
+# 4 Flow Diagrams
+
+## 4.1 FDB Flow Diagrams
+
+### 4.1.1 Local Static MAC upstream
+
+# 5 Error Handling
+
+# 6 Serviceability and Debug
+
+# 7 Warm Boot Support
+
+# 8 Scalability
+
+# 9 Limitations

```

Comment:
Kindly update this section with all known limitations.

Reply:
Limitations are captured in the DPB section